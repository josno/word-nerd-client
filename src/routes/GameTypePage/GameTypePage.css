.type-container {
	margin: 10px auto;
	width: 100%;
	display: flex;
	flex-direction: column;
	align-items: center;
	height: 600px;
}

.title-img-container {
	width: 150px;
	height: 150px;
	object-fit: contain;
	overflow: hidden;
	display: flex;
	align-items: center;
	justify-content: center;
	border: 1px solid black;
	border-radius: 5px;
	box-shadow: 4px 4px 10px grey;
}

.title-img {
	height: 100%;
}

@media (min-width: 800px) {
	.type-container {
		flex-direction: row;
		align-items: start;
		padding: 20px;
	}
}
