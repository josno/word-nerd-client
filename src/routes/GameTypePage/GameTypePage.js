import React, { useState, useEffect } from "react";

const GameTypePage = (props) => {
	return (
		<div>
			<h1>Select A Game</h1>
		</div>
	);
};

export default GameTypePage;
