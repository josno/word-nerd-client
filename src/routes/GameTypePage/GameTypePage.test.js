import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter as Router } from "react-router-dom";
import GameTypePage from "./GameTypePage";
import renderer from "react-test-renderer";

describe("<Login/>", () => {
	it("renders without crashing", () => {
		const div = document.createElement("div");
		ReactDOM.render(
			<Router>
				<GameTypePage />
			</Router>,
			div
		);
		ReactDOM.unmountComponentAtNode(div);
	});
	it("renders as expected", () => {
		const tree = renderer
			.create(
				<Router>
					<GameTypePage />
				</Router>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
