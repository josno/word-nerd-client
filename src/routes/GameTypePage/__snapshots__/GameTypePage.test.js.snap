// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> renders as expected 1`] = `
<div>
  <h1>
    Select A Game
  </h1>
  <div
    className="type-container"
  >
    <a
      href="/game//game-start-page"
      onClick={[Function]}
    >
      <div
        className="title-img-container"
      >
        <img
          alt="pass-the-ball"
          className="title-img"
          src="https://raw.githubusercontent.com/josno/word-nerd-client/master/assets/titlecards/PassTheBall_TitleCard.png"
        />
      </div>
    </a>
  </div>
</div>
`;
